index.html

```diff
@@ -186,7 +186,7 @@ <h4 style="margin-left: 20px;">Staff Members</h4>
           <h4>HimbeersaftLP - Admin Member</h4>
           <p>An amazing and hilarious developer who always contributes!</p>
           <ul>
-            <li><a href="https://goo.gl/9M3Xzs"><i class="fa fa-globe"></i></a></li>
+            <li><a href="https://himbeer.me"><i class="fa fa-globe"></i></a></li>
```

Comment:
That was the first thing I wanted to do after I saw the first commit, but luckily you were faster lol thanks